packages/widget/src/components/Routes/RoutesExpanded.tsx

```diff
@@ -43,13 +46,18 @@ export const RoutesExpanded = () => {
   const match = Boolean(element?.props?.children);
 
   return (
-    <CollapseContainer>
+    <CollapseContainer id="CollapseContainer">
```

Comment:
Let's add this id attribute here please to be consistent https://github.com/lifinance/widget/blob/main/packages/widget/src/utils/elements.ts#L1

Reply:
I think can remove these - I've accidently left them in from debug the routes positioning so they aren't really needed